pebblo/app/service/local_ui_service.py

```diff
@@ -399,67 +440,90 @@ def get_vector_dbs(chains: dict) -> list:
         return list(set(vector_dbs))
 
     @staticmethod
-    def get_all_documents(retrieval_data: dict) -> dict:
+    def sort_based_on_retrievals(retrieval_data: dict, search_key: str) -> dict:
```

Comment:
- `sort_retrievals` would make more sense here.
- `retrieval_data` would be `list`. Please confirm